lib/MyChartAPI.js

```diff
@@ -203,10 +207,53 @@ async function AddFutureWeeks(
     return results;
 }
 
+const CommonFilters = {
+    DaysOfWeek: {
+        0: true,
+        1: true,
+        2: true,
+        3: true,
+        4: true,
+        5: true,
+        6: true,
+    },
+    TimesOfDay: "both",
+
+    // these functions take a list of IDs and convert them to an object with the IDs as keys, and values all true.
+    Departments: (departments) =>
+        Object.fromEntries(departments.map((dept) => [dept, true])),
+    Providers: (providers) =>
+        Object.fromEntries(providers.map((dept) => [dept, true])),
+};
+
+function CommonPostDataCallback(providers, departments, vt, view = "grouped") {
+    const Departments = CommonFilters.Departments(departments);
+    const Providers = CommonFilters.Providers(providers);
```

Comment:
It seems dangerous to have a TitleCase const var and a camelCase parameter (function-scoped var) of the same name. Looks like you're using TitleCase for … actually, I'm not exactly sure what the rule is…Filters which are…object properties? idk, but confusion seems inevitable.

Reply:
filters need to end up in the form of `{"Departments":{...},"Providers":{...}}`, but url-encoded. Using the variables `Departments` and `Providers` lets me use the special javascript syntax of just dropping the variable in the object, such that the variable name becomes the key and variable value becomes the value ([here](https://github.com/livgust/covid-vaccine-scrapers/pull/122/files/b6b14a69a31f90c89e6a8b808619a03bb79bbddd#diff-dcf9415b8462bc0cba3fdacb5af869f64fe9e3f4168c28ad9e307e8999fdc7d6R236)). If you think that's problematic, I can definitely change it.

Reply:
[ I don't know why I didn't see your comment yesterday. I could swear I looked at this PR again more than 10 minutes after I left my comment and saw no response. Obviously my fault, sorry for the delay! ]

OK, I see, I definitely did not understand that this was conforming to the site API.
I don't want to hold this up any further. Consider adding a comment to clarify, maybe, but not necessary.
I'll approve.